src/mediarecording/WebMMediaRecording.ts

```diff
@@ -74,6 +77,33 @@ export default class WebMMediaRecording implements MediaRecording {
     });
   }
 
+  pause(): Promise<void> {
+    return new Promise((resolve, reject) => {
+      if (this.delegate === null) {
+        reject(new Error('not started'));
+      }
+      // this event should fire after any data is de-queued
+      this.delegate.addEventListener('pause', () => {
+        resolve();
+      });
+      this.delegate.pause();
+    });
+  }
+
+  unpause(): Promise<void> {
+    return new Promise((resolve, reject) => {
+      if (this.delegate === null) {
+        reject(new Error('not started'));
+      }
+      resolve();
+      this.delegate.resume();
+    });
+  }
+
+  get RecordingState(): string {
```

Comment:
`RecordingState` should be camelCase `recordingState`

Reply:
Also since there is a type for this, can we return `RecordingState` instead of `string`?